multisense_ros/src/reconfigure.cpp

```diff
@@ -80,6 +81,12 @@ Reconfigure::Reconfigure(Channel* driver,
     {
         motor_supported_ = true;
     }
+    if (system::DeviceInfo::HARDWARE_REV_MULTISENSE_S7AR == deviceInfo.hardwareRevision ||
```

Comment:
Is the s7ar supposed to be the s27

Reply:
Good catch. I saw a phantom 2 in-between the S and 7. Let me fix this